client/me/reauth-required/index.jsx

```diff
@@ -245,30 +275,33 @@ class ReauthRequired extends Component {
 		const hasSmsRecoveryNumber =
 			!! this.props?.twoStepAuthorization?.data?.two_step_sms_last_four?.length;
 
+		const currentAuthType = enhancedSecurity ? 'webauthn' : twoFactorAuthType;
```

Comment:
It would be better if `twoFactorAuthType` always had the right value. That can be achieved by:
1. Initializing the `twoFactorAuthType` state correctly in the component constructor. Now the initial value is hardcoded to `authenticator`, it should be `enhanced ? 'webauthn' : 'authenticator'`.
2. Not letting `TwoFactorActions` to switch it away from `webauth`. That probably already works.